webapp/views/App/views/Data/Charts/components/Panel/components/BlocksBuilder/BlocksBuilder.js

```diff
@@ -19,7 +26,16 @@ const BlocksBuilder = ({ dimensions, spec, onUpdateSpec }) => {
           {chartsConfig[configKey].selector.title}
         </button>
       ))}
-      <p>{type}</p>
+      {type &&
+        chartsConfig[type].builderBlocks.order.map((blockKey) =>
```

Comment:
can you extract chartsConfig[type].builderBlocks to a variabla, just to make it a bit more readable?